client/my-sites/plugins/plugin-details.jsx

```diff
@@ -353,12 +357,27 @@ function PluginDetails( props ) {
 										status="is-warning"
 										showDismiss={ false }
 									>
-										<NoticeAction href="https://wordpress.com/support/incompatible-plugins/">
+										<NoticeAction
+											href={ localizeUrl( 'https://wordpress.com/support/incompatible-plugins/' ) }
+										>
 											{ translate( 'More info' ) }
 										</NoticeAction>
 									</Notice>
 								) }
 
+								{ ! isMaintained && (
+									<Notice showDismiss={ false } status="is-warning">
+										{ translate(
+											'This plugin seems outdated and not maintained anymore. We advise that you {{a}}search{{/a}} for another similar plugin.',
```

Comment:
Nit: The language of the modal seemed too formal to me, maybe we can try something more direct?
```suggestion
											'This plugin seems outdated and not maintained anymore. Try {{a}}searching{{/a}} for a similar plugin.',
```

Reply:
Is there a reason we haven't used the .org copy?

> This plugin **hasn’t been tested with the latest 3 major releases of WordPress.** It may no longer be maintained or supported and may have compatibility issues when used with more recent versions of WordPress. Try {{a}}searching{{/a}} for a similar plugin.

Reply:
The message was discussed in the grooming https://github.com/Automattic/dotcom-forge/issues/3312#issuecomment-1658635571 but happy to change it to the same version than `.org`, @gcsecsey , do you agree with using the `.org` version?
>This plugin hasn’t been tested with the latest 3 major releases of WordPress. It may no longer be maintained or supported and may have compatibility issues when used with more recent versions of WordPress. Try {{a}}searching{{/a}} for a similar plugin.

Reply:
Yes I think the `.org` copy explains the situation best for the user, let's go with that one.

Reply:
Changed in https://github.com/Automattic/wp-calypso/pull/81091/commits/a6da7c5ff7494503d998b562ba9ae9701a147845